Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,30 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle.js';
+import HeaderContent from './HeaderContent.js';
+import ImageThumbnail from './ImageThumbnail';
+
+
+function HeaderContainer(props) {
+
+    return (
+        <>
+        <div className="header-container">
+            <div className="image-container">
+                <ImageThumbnail />
+            </div>
+            <div className="content-container">
```

Comment:
Try: 

something like: ```<div className="content-container top">Stuff here</div>```

Reply:
There are also too many divs here. Components themselves don't need divs since they have an outer div inside the actual component themselves. You are just nesting unnecessary divs at this point.